src/module.ts

```diff
@@ -39,25 +39,24 @@ export default defineNuxtModule<ModuleOptions>({
         analytics: options.analytics,
         cloud: options.cloud,
         url: options.url,
-      }
-    );
+      },
+    )
 
     if (!nuxt.options.runtimeConfig.public.cloudinary?.cloudName)
       console.warn(
-        "`[@nuxtjs/cloudinary]` Environment variable `CLOUDINARY_CLOUD_NAME` or property `cloudinary.cloudName` missing"
-      );
+        '`[@nuxtjs/cloudinary]` Environment variable `CLOUDINARY_CLOUD_NAME` or property `cloudinary.cloudName` missing',
+      )
 
-    const runtimeDir = fileURLToPath(new URL("./runtime", import.meta.url));
-    nuxt.options.build.transpile.push(runtimeDir);
+    const runtimeDir = fileURLToPath(new URL('./runtime', import.meta.url))
+    nuxt.options.build.transpile.push(runtimeDir)
 
-    addImportsDir(resolver.resolve(runtimeDir, "composables"));
+    addImportsDir(resolver.resolve(runtimeDir, 'composables'))
     addComponentsDir({
-      path: resolver.resolve(runtimeDir, "components"),
+      path: resolver.resolve(runtimeDir, 'components'),
       pathPrefix: false,
-      prefix: "",
-      // @ts-ignore
+      prefix: '',
       level: 999,
```

Comment:
```suggestion
      priority: 999,
```
You're probably looking for `priority` here

Reply:
Correct, added. Thanks!